.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Release Build
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v2
+      with:
+        java-version: 11
+        distribution: 'temurin'
```

Comment:
We should add Gradle caching, so that subsequent builds are faster:
```
- uses: actions/setup-java@v2
  with:
    java-version: '11'
    distribution: 'temurin'
    cache: 'gradle'
```